core/src/main/java/org/apache/accumulo/core/rpc/SslConnectionParams.java

```diff
@@ -264,7 +264,7 @@ public int hashCode() {
     }
     hash = 31 * hash + clientProtocol.hashCode();
     hash = 31 * hash + Arrays.hashCode(serverProtocols);
-    return super.hashCode();
+    return hash;
```

Comment:
I wonder if the original intent was something like `super.hashCode() + hash`, but they left it unfinished?

Reply:
Not sure, but I think this is what was breaking the caching at the ThriftTranportPool. The ThritTransportKey would get a different hash and create a new connection.

Reply:
No idea.